src/framebuffer.rs

```diff
@@ -29,8 +32,8 @@ pub const fn buffer_size<C: PixelColor>(width: usize, height: usize) -> usize {
 
 /// Calculates the required buffer size.
 ///
-/// This function is a workaround for current limitations in Rust const generics.
-/// It can be used to calculate the `N` parameter based on the size and bit depth of the framebuffer.
+/// This function is a workaround for current limitations in Rust const generics. It can be used to
+/// calculate the `N` parameter based on the size and bit depth of the framebuffer.
 pub const fn buffer_size_bpp(width: usize, height: usize, bpp: usize) -> usize {
```

Comment:
`buffer_size` and `buffer_size_bpp` will need a direction (type) parameter. See the `SetRawPixel` comment below.